Update internationalization.md (#5141)
flutter/website: src/docs/development/accessibility-and-localization/internationalization.md

    sdk: flutter         # Add this line
```

> Next, run `pub get packages` and import the flutter_localizations library and specify

Reviewer: ```suggestion
Next, run `pub get packages`, then import the `flutter_localizations` library and specify
```

`localizationsDelegates` and `supportedLocales` for `MaterialApp`:

<!-- skip -->
```dart
import 'package:flutter_localizations/flutter_localizations.dart';
// TODO: uncomment the line below after codegen
// import 'package:flutter_gen/gen_l10n/app_localizations.dart';

// ...

MaterialApp(
 localizationsDelegates: [
   // ... app-specific localization delegate[s] here

>    // TODO: uncomment the line below after codegen
>    //AppLocalizations.delegate,

Reviewer: ```suggestion
   // TODO: uncomment the line below after codegen
   // AppLocalizations.delegate,
```

   }
   ```

6. Now, run your app so that codegen takes place. You should see generated files in
   `${FLUTTER_PROJECT}/.dart_tool/flutter_gen/gen_l10n`.

> 7. Remove the comment for the import statement on `app_locations.dart` and `AppLocations.delegate` in
>    your call to the constructor for Material App. Test the generated localizations in your app as follows:

Reviewer: ```suggestion
7. Remove the comment for the import statement on `app_localizations.dart` 
   and `AppLocations.delegate` in your call to the constructor for 
   `MaterialApp`. Test the generated localizations in your app as follows:
```

   <!-- skip -->
   ```dart
   import 'package:flutter_localizations/flutter_localizations.dart';

>    import 'package:flutter_gen/gen_l10n/app_localizations.dart'; // remove the comment for this line

Reviewer: ```suggestion
   // import 'package:flutter_gen/gen_l10n/app_localizations.dart'; // remove the comment for this line
```

   // ...
   
   MaterialApp(
     localizationsDelegates: [
       // ... app-specific localization delegate[s] here

>        //AppLocalizations.delegate, // remove the comment for this line

Reviewer: ```suggestion
       // AppLocalizations.delegate, // remove the comment for this line
```
